include/aikido/util/yaml_utils.hpp

```diff
@@ -0,0 +1,202 @@
+#ifndef AIKIDO_UTIL_YAMLUTILS_HPP_
+#define AIKIDO_UTIL_YAMLUTILS_HPP_
+
+#include <functional>
+#include <mutex>
+#include <vector>
+#include <iostream>
+#include <boost/format.hpp>
+#include <boost/make_shared.hpp>
```

Comment:
Of these, I think only `Eigen/Dense`, `yaml-cpp/yaml.h`, and `boost/format.hpp` are actually used. We should remove the unused `#include` directives.

Actually, I'd also like to replace `boost::format` with `std:;stringstream` to eliminate that dependency as well.